core/src/test/java/com/crawljax/core/CandidateElementExtractorTest.java

```diff
@@ -179,6 +179,31 @@ public void whenFollowExternalUrlDoFollow() throws IOException, URISyntaxExcepti
 		assertThat(extract, hasSize(3));
 	}
 
+	@Test
+	public void testExtractContentSpecificFilenameProblem() throws Exception {
```

Comment:
The name should describe what's being tested, maybe`testExtractShouldIgnoreDownloadFiles`?